integration-tests/src/test/java/tachyon/client/TachyonFileUpdateIntegrationTest.java

```diff
@@ -44,6 +44,9 @@
 
   @Before
   public final void before() throws IOException {
+    // Disable hdfs client caching to avoid file system close() affecting other clients
+    System.setProperty("fs.hdfs.impl.disable.cache", "true");
```

Comment:
I tried to just keep with what we are currently using, I think it is a bit confusing to use before with beforeClass, I've opened a JIRA ticket to investigate the best practices: https://tachyon.atlassian.net/browse/TACHYON-585